pymoveit2/moveit2.py

```diff
@@ -23,9 +24,11 @@
     QoSReliabilityPolicy,
 )
 from sensor_msgs.msg import JointState
-from shape_msgs.msg import SolidPrimitive
+from shape_msgs.msg import SolidPrimitive, Mesh, MeshTriangle
 from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
 
+import trimesh
```

Comment:
Python module `trimesh` is great. However, not everyone using `pymoveit2` would require adding collision objects into the scene. Therefore, I would not make `trimesh` a required dependency. Instead, I would include something like this inside `add_collision_mesh()` to make it optional.

```py
try:
    import trimesh
except ImportError as err:
    raise ImportError(
        "Python module 'trimesh' no found! Please install it manually in order "
        "add collision object into the MoveIt 2 planning scene."
    ) from err
```